### DISCUSSION TITLE
Question how or where do you generate the token for the administration api

### DISCUSSION THREAD

**Discussion by PaulMaur** [AzuraCast/AzuraCast#4305] [Q&A] [answered]
**Using Docker installation method?**
Yes 

**AzuraCast version:**
AzuraCast • Rolling Release #133a943 (2021-06-16 19:34) • Docker • PHP 8.0 

**Host Operating System:**
debian 10

**question**
Question how or where do you generate the token for the administration api

**Comment by AaronUnmuted** [comment, marked as answer]
http://yourinstallation.url/api_keys
